merge_mafs/merge_mafs.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import argparse
+
+parser = argparse.ArgumentParser(description = 'This program reads in 2 MAF files and merges them to one.')
+parser.add_argument("maf_file1", type=str, help="Input MAF1")
+parser.add_argument("maf_file2", type=str, help="Input MAF2")
+parser.add_argument("merged_maf", type=str, help="Output merged MAF filename")
+args = parser.parse_args()
+
+file_header = list();
+
+# Check if columns in the two input maf files are in same order	
+
+try:
+	with open(args.maf_file1,'r') as file:
+		header = file.readline()
```

Comment:
headers may sometimes contain comments (i.e., VCF2MAF version, #sequenced_samples, etc.). I would first check whether the line starts with `#` (metadata prefix) and skip until you hit a line that does not start with `#`

Reply:
adding onto this -- we may have some functions that pull out the first non-# line in a file in the clinicalfile_utils python module. Maybe we can use stuff from there (or if it's a new function, add it there)